components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -1371,12 +1371,12 @@ public void testGetUser(Boolean isGroupsVsRolesSeparationImprovementsEnabled, Ma
         when(mockedSCIMGroupHandler.listSCIMRoles()).thenReturn(scimRoles);
         when(mockedSCIMGroupHandler.getGroupWithAttributes(any(Group.class), anyString()))
                 .thenAnswer(new Answer<Group>() {
-            public Group answer(InvocationOnMock invocation) throws Throwable {
-                Object[] args = invocation.getArguments();
-                String groupName = (String) args[1];
-                return groupsMap.get(groupName);
-            }
-        });
+                    public Group answer(InvocationOnMock invocation) throws Throwable {
+                        Object[] args = invocation.getArguments();
+                        String groupName = (String) args[1];
+                        return groupsMap.get(groupName);
+                    }
+                });
```

Comment:
Shall we revert these changes since they are not related to this fix?

Reply:
Fixed : [1c24b8ec2f4961c](https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/451/commits/1c24b8ec2f4961c6cfa67fc07257a87c350adc19)